src/Request.php

```diff
@@ -307,18 +315,28 @@ public function getPost($name = null, $default = null)
             return $this->postParams;
         }
 
+        trigger_error(
+            'Getting specific post parameter is deprecated',
+            E_USER_DEPRECATED
+        );
+
         return $this->postParams->get($name, $default);
     }
 
     /**
      * Return the Cookie header, this is the same as calling $request->getHeaders()->get('Cookie');
      *
      * @convenience $request->getHeaders()->get('Cookie');
-     * @return Header\Cookie|bool
+     * @return Header\Cookie|false
```

Comment:
`false` is not a type. Generally look for all `|false` or `false|` in your patch and replace them with `bool`

Reply:
`false` keyword is accept by phpdoc